src/overlays/actors/ovl_Bg_Relay_Objects/z_bg_relay_objects.c

```diff
@@ -10,10 +10,7 @@
 
 #define THIS ((BgRelayObjects*)thisx)
 
-typedef enum {
-    WINDMILL_ROTATING_GEAR,
-    WINDMILL_DAMPE_STONE_DOOR
-} WindmillSetpiecesMode;
+typedef enum { WINDMILL_ROTATING_GEAR, WINDMILL_DAMPE_STONE_DOOR } WindmillSetpiecesMode;
 
```

Comment:
i know this wasnt your change, but can you add offset comments so it doesnt get memed on from clang